osc_sdk/sdk.py

```diff
@@ -636,7 +673,12 @@ def make_request(self, action, *args, **kwargs):
                 signed_headers,
             )
 
-        res = requests.request(
+        request = requests.Session()
```

Comment:
this seems a duplication of what's at line 524

Reply:
The whole make_request function is pretty much a duplication from its mother class which already existed prior to this branch.
This would need a separate refactoring commit indeed.

Reply:
Refactoring done as a separate commit